src/LanguageServer.ts

```diff
@@ -893,16 +893,6 @@ export class LanguageServer {
 
         //return the first non-falsey hover. TODO is there a way to handle multiple hover results?
         let hover = hovers.filter((x) => !!x)[0];
-
-        //TODO improve this to support more than just .brs files
```

Comment:
This was forcing all hovers to be brightscript...but we need to support any documentation information provided, so force the underlying files to provide the full markdown text instead of casting it all here.